SalsaCompiler/SalsaCompiler/main.swift

```diff
@@ -52,80 +52,89 @@ let exportFile: URL = {
 // Find a name for the working directory
 let workingDirectory: String = {
   while true {
-    let directory = "/tmp/Salsa/\(UUID().uuidString)"
+    let directory = "\(NSTemporaryDirectory())\(UUID().uuidString)"
```

Comment:
I think we want a subdirectory of the `NSTemporaryDirectory`, right? If so there's a slash missing.

Could use more of the path APIs, like: `NSTemporaryDirectory().appendingPathComponent(UUID().uuidString)`

Reply:
NSTemporaryDirectory() ends with a / which is why I omitted it. But I like doing `appendingPathComponent` better